kordac/tests/ScratchTest.py

```diff
@@ -34,6 +45,8 @@ def test_doc_example_basic(self):
         self.assertSetEqual(actual, expected)
```

Comment:
and therefore ``expected`` should probably also be a little bit more descriptive